### PR TITLE
Fix copy_quizzes to use `quiz_dir`

### PR DESCRIPTION
**Pull request by cansavvy**
#### What changes are being implemented in this Pull Request?

This is the sequel to #80 because I reviewed it too quickly and it didn't copy the quizzes. :sad: 

But now it does. I changed it to take the quiz_dir argument that a user specifies in the `bookdown_to_leanpub()` main function so it should inherit whatever that value is. Also added a message for if that directory isn't found.

### CODE REVIEW

File: R/bookdown_to_leanpub.R [hunk 1]
```diff
@@ -146,11 +146,14 @@ copy_bib <- function(path = ".", output_dir = "manuscript") {
...
-  path <- bookdown_path(path)
-  files <- list.files(path = path, full.names = TRUE, pattern = "^quiz")
+copy_quizzes <- function(quiz_dir = "quizzes", output_dir = "manuscript") {
+  if (!dir.exists(quiz_dir)) {
+    stop(paste("The quiz directory specified by quiz_dir:", quiz_dir, "does not exist."))
+  }
+  quizzes <- list.files(path = quiz_dir, full.names = TRUE, pattern = "\\.md$")
   if (length(files) > 0) {
-    file.copy(files, file.path(output_dir), overwrite = TRUE)
+    fs::file_copy(quizzes, file.path(output_dir, basename(quizzes)),
```
Comment (cansavvy) [inline R/bookdown_to_leanpub.R:155]: It copies over the files, ditches the directory. Is this what we want, @carriewright11 ?

**Comment by cansavvy** [comment]
Now to see if the checks pass.

File: R/bookdown_to_leanpub.R (hunk 1 above)
Comment (carriewright11) [reply R/bookdown_to_leanpub.R:155]: yeah I think that makes sense

**Review by carriewright11 — APPROVED** [review]

File: R/bookdown_to_leanpub.R (hunk 1 above)
Comment (carriewright11) [reply R/bookdown_to_leanpub.R:155]: ah yes... good point about the directory...

**Pull request merged**